test/01-constrain.test.ts

```diff
@@ -0,0 +1,78 @@
+import * as supertest from 'supertest';
+import { expect } from 'chai';
+const fixturePath = __dirname + '/fixtures/01-constrain/config';
+import { testInit, testDeInit, testLocalServer } from './lib/test-init';
+
+describe('01 basic constrain tests', function () {
+	let pineServer: any;
+	before(async () => {
+		pineServer = await testInit(fixturePath, true);
+	});
+
+	describe('Basic', () => {
+		it('check /ping route is OK', async () => {
+			const res = await supertest(testLocalServer).get('/ping').expect(200);
+			expect(res.text).to.equal('OK');
```

Comment:
I think you can write this as
```suggestion
			const res = await supertest(testLocalServer).get('/ping').expect(200, 'OK');
```
although my `supertest` api knowledge isn't that intimate so I might be wrong